blueprints/grafana/dashboard_group.libsonnet

```diff
@@ -12,7 +12,9 @@ function(policyJSON, cfg) {
       title: 'Aperture Signals - %s' % policyName,
     },
   }).dashboard,
+  local receiverDashboards = creator(policyJSON, cfg).receiverDashboards,
```

Comment:
extract output from creator to a separate var then from there extract dashboards - why run the creator twice?